pwa/pages/books/[id]/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { NextComponentType, NextPageContext } from "next";
+import { Show } from "../../../components/book/Show";
+import { Book } from "../../../types/Book";
+import { fetch } from "../../../utils/dataAccess";
```

Comment:
note to self the path needs aliasing and what is that directory name :laughing:

Reply:
Those changes must be done in the distribution, it will be backported to the demo then

Reply:
[Path aliasing](https://nextjs.org/docs/advanced-features/module-path-aliases) added